src/main/webapp/login-script.js

```diff
@@ -48,6 +51,25 @@ async function signInPrivate(googleUser) {
   }
 }
 
+async function signInReports(googleUser) {
```

Comment:
Nit: I feel this would be a bit clearer if it were "reportsSignIn" perhaps

Reply:
Makes sense, changed signInPrivate to fit this convention too